tests/medium/test_games.py

```diff
@@ -53,6 +53,20 @@ def setUp(self):
         self.installed_path = os.path.join(self.install_base_path, "games", "blender")
 
 
+class RenPyInContainer(ContainerTests, test_games.RenPyTests):
+    """This will test the Ren'Py editor inside a container"""
+
+    TIMEOUT_START = 20
+    TIMEOUT_STOP = 10
+
+    def setUp(self):
+        self.hosts = {443: ["www.renpy.org"]}
+        self.apt_repo_override_path = os.path.join(self.APT_FAKE_REPO_PATH, 'renpy')
```

Comment:
This is not needed since renpy apparently doesn't have other dependencies than need to be installed (is that correct?)